src/main/java/com/company/managementservice/service/OrganisationService.java

```diff
@@ -0,0 +1,63 @@
+package com.company.managementservice.service;
+
+
+
+import com.company.managementservice.exception.NotFoundException;
+import com.company.managementservice.model.dto.DepartmentDto;
+import com.company.managementservice.model.dto.OrganisationDto;
+import com.company.managementservice.model.entity.Department;
+import com.company.managementservice.model.entity.Organisation;
+import com.company.managementservice.repo.OrganisationRepo;
+import org.modelmapper.ModelMapper;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Component;
+
+import java.util.Optional;
+
+@Component
+public class OrganisationService {
+
+    @Autowired
+    OrganisationRepo repo;
+
+    private ModelMapper modelMapper = new ModelMapper();
+
+    public OrganisationDto saveOrganisation(OrganisationDto organisationDto) {
+
+        Organisation organisation=repo.save(modelMapper.map(organisationDto, Organisation.class));
+        organisationDto.setId(organisation.getId());
+        return modelMapper.map(organisation,OrganisationDto.class);
+
+    }
+
+    public OrganisationDto getOrganisation(Integer id) throws NotFoundException {
```

Comment:
final @NonNull Integer id in parameter. Add these everywhere where parameters are being passed